.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+Please follow the instructions and template below. Otherwise, to maintain quality of issues we may be forced to close your issue.
```

Comment:
>Otherwise, to maintain quality of issues we may be forced to close your issue. [](start = 51, length = 78)

I can't imagine us closing an issue on someone in practice.  Could we reword the "Otherwise, to maintain quality...' to something like

"Otherwise, we may have to spend additional time requesting this information from you and your resolution time may be delayed."